app/src/main/java/edu/temple/templetag/HomeActivity.java

```diff
@@ -237,35 +250,50 @@ public void run() {
                         @Override
                         public void onComplete(@NonNull Task<QuerySnapshot> task) {
                             if (task.isSuccessful()) {
+                                Tags.clear();
```

Comment:
What does Tags.clear() do?

Reply:
It clears out the list of tags every time we fetch new tags